internal/provider/telemetry_resource.go

```diff
@@ -129,6 +148,12 @@ func (r *TelemetryResource) Create(ctx context.Context, req resource.CreateReque
 
 	newId := uuid.NewString()
 	data.Id = types.StringValue(newId)
+	if data.Nonce.IsUnknown() {
```

Comment:
These values are marked as read-only (computed), so they will always be unknown I think.

Also could we set them to null rather than zero? types.NumberNull(). This must get depend on legacy modules requiring a value here.

Reply:
Sorry my mistake. Optional, Computed allows the value to be set by the caller but also overridden.

Should we set null here though?

Reply:
```go
if data.Nonce.IsUnknown() {
		data.Nonce = types.NumberNull()
	}
	if data.EphemeralNumber.IsUnknown() {
		data.EphemeralNumber = types.NumberNull()
	}
```

I think the default value for `Unknown` is already `null` here @matt-FFFFFF ?